footer.php

```diff
@@ -49,7 +50,7 @@
 			</div>
 
 			<div class="cell" style="margin-bottom: 50px;">
-				<div class="site-info text-center"><?php base_theme_copyright_text(); ?></div><!-- .site-info -->
+				<div class="site-info text-center"><?php echo _e( $base_theme_footer_copyright ); ?></div><!-- .site-info -->
```

Comment:
@rashmi-sc Have you checked the documentation for the `_e` function? Please correct this.

Reply:
done